projects/ds/README.md

```diff
@@ -1,11 +1,28 @@
-# Ds
-
-This library was generated with [Angular CLI](https://github.com/angular/angular-cli) version 12.0.5.
+# DS - Radius Angular Design System Starter
+
+This starter component library was generated with [Angular CLI](https://github.com/angular/angular-cli) version 12.0.5.
+
+What's inside:
+
+```
+.
+├── src
+│   ├── docs
+│   │   ├── static
+│   │   │   └── imgs
+│   │   └── tokens
+│   └── lib
+│       ├── button
+│       ├── tag
+│       └── text
+└── styles
```

Comment:
Nice!

Reply:
Yes, `tree` tool changed the game for me! actually one of my fave README's was from a Gatsby starter because it situated me so quickly https://github.com/wataruoguchi/gatsby-starter-typescript-contentful